avocado/core/remote/runner.py

```diff
@@ -86,7 +89,13 @@ def check_remote_avocado(self):
         if result.exit_status == 127:
             return (False, None)
 
+        # Get the remote and local versions of avocado, and compare the major versions only
         match = self.remote_version_re.findall(result.stdout)
+        self.remote_version = '.'.join(list(match[0]))
+        self.local_version = version.VERSION
+        if self.remote_version.split('.')[0] != self.local_version.split('.')[0]:
```

Comment:
Why to join `self.remote_version` to then split? If you join it only for the log message (after this `if`) you don't need to split.